fdm-docs/docs/balance/01-nitrogen.md

```
@@ -191,13 +191,85 @@ Crop residues (e.g., straw, stover, roots) contain significant amounts of nitrog
+
+    *   **Formula:**
+        `NH3 Emission (kg N / ha) = Application Amount (kg / ha) * Total N Content (fraction) * Emission Factor (fraction)`
+        Where:
+        *   `Application Amount`: `p_app_amount` (kg / ha).
+        *   `Total N Content`: `p_n_rt` (fraction).
+        *   `Emission Factor`: `p_ef_nh3` (fraction). This factor can be directly provided in the `FertilizerDetail`. If it is not provided, it is calculated using an empirical formula based on the fertilizer's composition:
+
+        `Emission Factor = p_n_org^2 * K_1 + p_no3_rt * p_s_rt * K_2 + p_nh4_rt^2 * K_3`
+
+        Where:
+        *   `p_n_org`: Organic nitrogen content (calculated as `p_n_rt - p_no3_rt - p_nh4_rt`).
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 4; Actual: 8

(MD007, ul-indent)